### ISSUE TITLE
[Audio Player]: QA: No label for "Rewind/Forward" slider control and the Volume slider control

### ISSUE THREAD

**Issue by gopinathan-n** [carbon-design-system/carbon-for-ibm-dotcom#9280]
### Description

Page Element: Rewind/Forward slider control and the Volume slider control

Issue Description:
No label for "Rewind/Forward" slider control and the Volume slider control

Expected:
Provide appropriate label

User Impact:
When the screen reader is unable to identify a label, it’s difficult for users to understand the purpose of the control.

### Component(s) impacted

Audio Player

### Browser

Safari, Firefox

### Carbon for IBM.com version

v1.37.0-rc.0

### Severity

Severity 2 = Aspects of design is broken, and impedes users in a significant way, but there is a way to complete their tasks. Affects major functionality, has a workaround.

### Application/website

Carbon for IBM.com

### Package

@carbon/ibmdotcom-web-components

### CodeSandbox example

https://carbon-design-system.github.io/carbon-for-ibm-dotcom/canary/react-experimental/?path=/story/components-audio-player--default

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues
